docs/data/data-grid/events/CatalogOfEventsNoSnap.js

```diff
@@ -43,102 +58,96 @@ apiRef.current.subscribeEvent(
-            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
-          </IconButton>
-        </TableCell>
-        <TableCell style={{ borderBottom: 'unset' }}>
-          <code>{event.name}</code>
-          {planBadge}
-        </TableCell>
-        <TableCell style={{ borderBottom: 'unset' }}>
-          <div
-            dangerouslySetInnerHTML={{
-              __html: event.description,
-            }}
-          />
-          {!!event.params && (
-            <div>
-              <Typography variant="subtitle2" component="span">
-                Params:{' '}
-              </Typography>
-              <code
-                dangerouslySetInnerHTML={{
-                  __html: escapeHTML(event.params),
-                }}
-              />
-            </div>
-          )}
-          {!!event.event && (
-            <div>
-              <Typography variant="subtitle2" component="span">
-                Event:{' '}
-              </Typography>
-              <code
-                dangerouslySetInnerHTML={{
-                  __html: escapeHTML(event.event),
-                }}
-              />
-            </div>
-          )}
-        </TableCell>
-      </TableRow>
-      <TableRow>
-        <TableCell style={{ paddingBottom: 0, paddingTop: 0 }} colSpan={3}>
-          <Collapse in={open} timeout="auto" unmountOnExit>
-            <Box sx={{ margin: 1 }}>
-              <HighlightedCode code={example} language="tsx" />
-            </Box>
-          </Collapse>
-        </TableCell>
-      </TableRow>
-    </React.Fragment>
+    <Box sx={{ px: 1 }}>
+      <HighlightedCode code={example} language="tsx" />
+    </Box>
   );
 };
 
+const COLUMNS = [
+  {
+    field: 'name',
+    headerName: 'Name',
+    width: 240,
+  },
+  {
+    field: 'plan',
+    headerName: 'Plan',
+    width: 100,
+    align: 'center',
+    valueGetter: ({ row }) => {
+      if (row.projects.includes('x-data-grid')) {
+        return 'x-data-grid';
+      }
+      if (row.projects.includes('x-data-grid-pro')) {
+        return 'x-data-grid-pro';
+      }
+      if (row.projects.includes('x-data-grid-premium')) {
+        return 'x-data-grid-premium';
+      }
+      return null;
+    },
+    renderCell: ({ value }) => {
+      if (value === 'x-data-grid-pro') {
+        return <span className="plan-pro" title="Pro plan" />;
+      }
+      if (value === 'x-data-grid-premium') {
+        return <span className="plan-premium" title="Premium plan" />;
+      }
+      return '';
+    },
+  },
+  {
+    field: 'description',
+    headerName: 'Description',
+    flex: 1,
+    renderCell: ({ value, colDef }) => {
+      const width = colDef.computedWidth - 20;
+      return (
+        <Description
+          style={{ minWidth: width, maxWidth: width, width, whiteSpace: 'normal' }}
+          dangerouslySetInnerHTML={{
+            __html: value,
+          }}
+        />
+      );
+    },
+  },
+];
+
+const Toolbar = () => (
+  <GridToolbarContainer>
```

Comment:
Btw, on the [new demo](https://deploy-preview-5292--material-ui-x.netlify.app/x/react-data-grid/demo/), I added padding to the Toolbar container. It seemed a little better with the quick-filter alone.

```suggestion
  <GridToolbarContainer sx={{ p: 1 }}>
```